.circleci/config.yml

```diff
@@ -1269,7 +1269,7 @@ workflows:
     jobs:
       - spm-revenuecat-ui-ios-15
       - spm-revenuecat-ui-ios-16
```

Comment:
I do wonder if we should also rename these, if they are doing the same thing but on different iOS versions...